src/applications/proxy-rewrite/sass/style-consolidated.scss

```diff
@@ -121,7 +121,6 @@ $font-stack: "Source Sans Pro", "Helvetica Neue", "Helvetica", "Roboto", "Arial"
 
   @import "../../../platform/site-wide/sass/modules/m-footer";
   @import "../../../platform/site-wide/sass/modules/m-crisis-line";
-  @import "../../../platform/site-wide/sass/modules/m-search";
```

Comment:
I dunno if we should make changes to `style-consolidated`. it's used on the teamsite pages and I'm not 100% sure what any of this does :|

Reply:
Now that I have some more awareness of teamsite, I won't merge this just yet.